.github/CONTRIBUTING.md

```diff
@@ -126,53 +129,67 @@ If you want to test a branch before releasing it, you can push up a _built branc
    ```
 
 ## Git workflow
+### Commit style
 
-### Branches
-
-The only "special" branch is the `master` branch which represents the latest stable version of the library. Most PRs with bug fixes, features, and even breaking changes should probably be made against, and ultimately merged into, `master`. The only exception is if you're unsure of changes and need to validate your ideas in production before making an official stable Maker release, which would be most common in the case of breaking changes and major version releases, so let's go over an example of how we'd do one.
+This repository uses [Conventional Commits](https://www.conventionalcommits.org) for simple yet meaningful commit messages. Not only are they user-friendly, they are also machine-readable for automated release notes and versioning.
 
-Let's say Maker is on version `1.5.2` and you have a breaking change, or maybe several, that you'd like to make and that would require a major version release of `2.0.0`. Unless you're very confident in your changes you'll likely want to do a few _pre-releases_ first, and hold out on directly merging your changes directly into `master`. So what you'd do is:
 
-1. Make a `v2` branch, or whatever you want to call it, as long as the name is descriptive.
-2. Within this branch bump the version of Maker within `package.json` from `1.5.2` to `2.0.0-beta.0`.
-3. Make and merge all your tentative changes into `v2` and bump the pre-release version with every merged change, e.g. `-beta.0`, `-beta.1`, `-beta.2`, etc.
-4. Build, i.e. `npm run build`, and publish them to npm, i.e. `npm run publish`.
-5. Build and deploy the styleguide for your changes to get feedback from coworkers, i.e. `npm run styleguide-build`.
-6. Repeat steps 3 - 6 until you feel confident in your changes and they have stabilized.
-7. Open `v2` PR against `master`, get approval, merge in, build & publish official `2.0.0` release.
+It has the following formats:
 
-### Commit style
+#### Without scope
 
-Commit following the [Conventional Commits style](https://www.conventionalcommits.org) for simple yet meaningful commit messages. It has the following formats:
+```
+<type>: <subject>
+```
 
 #### With scope
-
 ```
 <type>(<scope>): <subject>
 ```
 
-#### Without scope
+#### Breaking change
 
 ```
-<type>: <subject>
+<type>(<scope>)!: <subject>
 ```
 
-Reference [this JSON](https://github.com/commitizen/conventional-commit-types/blob/master/index.json) for supported commit types.
+#### Breaking change with message
+
+```
+<type>: <subject>
+BREAKING CHANGE: Required message
+```
+
+#### Types
+- `feat`: new feature for the user, not a new feature for build script
+- `fix`: bug fix for the user, not a fix to a build script
+- `docs`: changes to the documentation
+- `build`: changes that affect the build system or external dependencies
```

Comment:
I'm curious what happens with this prefix, because a change that affects the build system affects how the production components are built which could be a patch change, or a minor change, or major change... so how does semantic release handle this particular prefix in regards to determining how to bump the package's version?

Reply:
If deciding between `feat` or `fix` vs `build`, choose `feat` or `fix` because they influence the version bump appropriately.

If neither but it's a breaking change, use `build!`.

Reply:
Those are good tips it'd be nice if they were documented :)